### PR TITLE
Support for End to end SSL and SSL Policy in application gateway

### PR DESCRIPTION
**Pull request by vinatara**

### CODE REVIEW

File: arm-network/2016-06-01/swagger/network.json [hunk 1]
```diff
@@ -4288,6 +4288,27 @@
       },
       "description": "SKU of application gateway"
     },
+    "ApplicationGatewaySslPolicy": {
+      "properties": {
+        "disabledSslProtocols": {
+          "type": "array",
+          "description": "Get or sets SSL protocols to be disabled on Application Gateway",
```
Comment (John-Hart) [inline arm-network/2016-06-01/swagger/network.json]: Please remove Get or Set from the description of all the model properties. Autorest will add it automatically as needed.

**Comment by hovsepm** [comment]
cc @DeepakRajendranMsft - please sign off

**Comment by DeepakRajendranMsft** [comment]
Thanks for removing "Gets or sets" LGTM!

**Comment by DeepakRajendranMsft** [comment]
LGTM after the usages list changes

**Pull request merged**